apps/docs/components/Breadcrumbs.tsx

```diff
@@ -0,0 +1,146 @@
+        <BreadcrumbItem>
+          {breadcrumbs[0].url ? (
+            <BreadcrumbLink href={`/docs${breadcrumbs[0].url}`}>
+              {breadcrumbs[0].title || breadcrumbs[0].name}
+            </BreadcrumbLink>
+          ) : (
+            <BreadcrumbPage>{breadcrumbs[0].title || breadcrumbs[0].name}</BreadcrumbPage>
+          )}
+        </BreadcrumbItem>
+        <BreadcrumbSeparator />
+        {breadcrumbs.length > ITEMS_TO_DISPLAY && (
+          <>
+            <BreadcrumbItem>
+              {!isMobile ? (
+                <DropdownMenu open={open} onOpenChange={setOpen}>
+                  <DropdownMenuTrigger className="flex items-center gap-1" aria-label="Toggle menu">
+                    <BreadcrumbEllipsis className="h-4 w-4" />
+                  </DropdownMenuTrigger>
+                  <DropdownMenuContent align="start">
+                    {breadcrumbs.slice(1, -2).map((crumb, index) => (
+                      <DropdownMenuItem
+                        key={index}
+                        className={cn(!crumb.url && 'pointer-events-none')}
+                      >
+                        {crumb.url ? (
+                          <Link href={`/docs${crumb.url}`}>{crumb.title || crumb.name}</Link>
+                        ) : (
+                          crumb.title || crumb.name
+                        )}
+                      </DropdownMenuItem>
+                    ))}
+                  </DropdownMenuContent>
+                </DropdownMenu>
+              ) : (
+                <Drawer open={open} onOpenChange={setOpen}>
+                  <DrawerTrigger aria-label="Toggle Menu">
+                    <BreadcrumbEllipsis className="h-4 w-4" />
+                  </DrawerTrigger>
+                  <DrawerContent showHandle={false}>
+                    <div className="grid gap-1 px-4">
+                      {breadcrumbs
+                        .slice(1, -2)
+                        .map((crumb) =>
+                          crumb.url ? (
+                            <Link href={`/docs${crumb.url}`}>{crumb.title || crumb.name}</Link>
+                          ) : (
+                            crumb.title || crumb.name
+                          )
+                        )}
+                    </div>
+                    <DrawerFooter className="pt-4">
+                      <DrawerClose asChild>
+                        <Button type="outline">Close</Button>
+                      </DrawerClose>
+                    </DrawerFooter>
+                  </DrawerContent>
+                </Drawer>
+              )}
+            </BreadcrumbItem>
+            <BreadcrumbSeparator />
+          </>
+        )}
+        {breadcrumbs?.slice(-ITEMS_TO_DISPLAY + 1).map((crumb, i) => (
+          <Fragment key={crumb.url}>
+            {i !== 0 && <BreadcrumbSeparator />}
+            <BreadcrumbItem className="flex items-center overflow-hidden last:text-foreground">
```

Comment:
@MildTomato don't you think this draws too much focus on the last breadcrumb itself?
(which also highlights the redundancy a bit too much imo)

<img width="1116" alt="Screenshot 2024-07-11 at 16 46 20" src="https://github.com/supabase/supabase/assets/25671831/beea7bf3-7e32-4bfb-9f86-4d47be46bf36">

Reply:
I've seen it both ways. Although the trend I see is last item has most dominance.
<img width="1551" alt="Screenshot 2024-07-11 at 7 07 13 PM" src="https://github.com/supabase/supabase/assets/8291514/0f909dff-7837-4b07-8296-d46a290c3ecc">

Reply:
I think it makes more sense if it's not close to the title of the page.
But lets go for dominance if you think that's the way 👍

Reply:
but using `foreground-light` seems like a good compromise 👍

Reply:
<img width="1142" alt="Screenshot 2024-07-11 at 19 54 12" src="https://github.com/supabase/supabase/assets/25671831/4c782e3e-bcad-40c6-8b85-a23a50586aef">